boards/av/x-v1/default.cmake

```diff
@@ -12,7 +12,7 @@ px4_add_board(
 	SERIAL_PORTS
 		GPS1:/dev/ttyS6
 		TEL1:/dev/ttyS0
-		TEL2:/dev/ttyS1
+		TEL2:/dev/ttyS5
```

Comment:
Could you explain the numbering change? Since the ports are unlabelled I was trying to keep the numbering consistent.